README.md

```diff
@@ -6,16 +6,17 @@ Wraps the inline content with `ins` and `del` tags.
 
 Installation
 ------------
-
-    pip install git+git://github.com/aleray/mdx_del_ins.git
-
+    
+    copy mdx_del_ins_markdown_3_0_1.py as a module
 
 Usage
 -----
 
     >>> import markdown
+    >>> from mdx_del_ins_markdown_3_0_1 import DelInsExtension
+
     >>> src = """This is ++added content++ and this is ~~deleted content~~""" 
-    >>> html = markdown.markdown(src, ['del_ins'])
+    >>> html = markdown.markdown(src, extensions=[DelInsExtension()])
```

Comment:
Similarly to the loss of installability via pip this makes configurations more complicated.